src/App.js

```diff
@@ -10,9 +18,17 @@ class App extends Component {
 
     render() {
         return (
-            <main role="main">
-                <MainPage />
-            </main>
+          <Router>
+            <Header />
+            <Switch>
+              <Route exact path='/'><Redirect to='/home' /></Route>
```

Comment:
Can we collapse the `/` and `/home` routes to just `/`? Thanks

Reply:
Done